src/layout/AttachmentList/AttachmentListComponent.tsx

```diff
@@ -41,7 +41,7 @@ export function AttachmentListComponent(props: IAttachmentListProps) {
         className='attachmentList-title'
         component='span'
       >
-        {props.text || ''}
+        {text || ''}
```

Comment:
I think we don't need to fallback to an empty string here, because null or undefined should not be rendered. 🤔 
```suggestion
        {text}
```